frontend/src/pages/Admin/Components/Faq/ManageFaq/manage-faq.module.scss

```diff
@@ -0,0 +1,56 @@
+.faq {
+    width: 80%;
+    margin: 0em auto 1em auto;
+    display: flex;
+    flex-direction: column;
+    height: 33rem;
```

Comment:
can we please use some more height as a lot of empty spaces are left un-necessarily